addons/web/static/src/js/views/kanban/kanban_controller.js

```diff
@@ -520,9 +521,11 @@ var KanbanController = BasicController.extend({
               this.model.actionArchive(recordIds, column.db_id) :
               this.model.actionUnarchive(recordIds, column.db_id);
             prom.then(function (dbID) {
-                var data = self.model.get(dbID);
+                let data = self.model.get(dbID);
                 if (data) {  // Could be null if a wizard is returned for example
                     self.model.reload(self.handle).then(function () {
+                        // Retrieve fresher data as the reload may have changed it.
+                        data = self.model.get(dbID);
```

Comment:
This was added because when archiving all the records of a active-filtered column, the progressbar (and the load more [remaining]) was not updated.